app/equipment.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any
```

Comment:
The import statement `from typing import Any` is correct, but the function `knights_parameters` does not use the `Any` type effectively. Consider using a more specific type annotation for `knight_name`, such as `dict` or a custom class, to improve code readability and type safety.